src/repr/src/lib.rs

```diff
@@ -28,6 +28,7 @@ mod scalar;
 
 pub mod adt;
 pub mod strconv;
+pub mod util;
```

Comment:
[minor nit / rant; feel free to ignore]

Idk how you feel about this but imo `util` crates / files end up one of two ways after a year or two

a) same size as the original
b) accrued a bunch of random stuff, now unclear what all is in `util` and its a source of tech debt

I don't have a good suggestion for a different name (maybe `arg_duration`?) but would ask you to consider alternatives.